DQMOffline/L1Trigger/python/LegacyL1TriggerDqmOffline_cff.py

```diff
@@ -0,0 +1,252 @@
+import FWCore.ParameterSet.Config as cms
+
+# L1 Trigger DQM sequence for offline DQM
+#
+# used by DQM GUI: DQM/Configuration 
+#
+#
+#
+# standard RawToDigi sequence and RECO sequence must be run before the L1 Trigger modules, 
+# labels from the standard sequence are used as default for the L1 Trigger DQM modules
+#
+# V.M. Ghete - HEPHY Vienna - 2011-01-02 
+#                       
+                      
+
+#
+# DQM L1 Trigger in offline environment
+#
+
+import DQMServices.Components.DQMEnvironment_cfi
+dqmEnvL1T = DQMServices.Components.DQMEnvironment_cfi.dqmEnv.clone()
+dqmEnvL1T.subSystemFolder = 'L1T'
+
+# DQM online L1 Trigger modules, with offline configuration 
+from DQMOffline.L1Trigger.L1TMonitorOffline_cff import *
+from DQMOffline.L1Trigger.L1TMonitorClientOffline_cff import *
+
+
+# DQM offline L1 Trigger versus Reco modules
+
+import DQMServices.Components.DQMEnvironment_cfi
+dqmEnvL1TriggerReco = DQMServices.Components.DQMEnvironment_cfi.dqmEnv.clone()
+dqmEnvL1TriggerReco.subSystemFolder = 'L1T/L1TriggerVsReco'
+
+#
+# DQM L1 Trigger Emulator in offline environment
+# Run also the L1HwVal producers (L1 Trigger emulators)
+#
+
+import DQMServices.Components.DQMEnvironment_cfi
+dqmEnvL1TEMU = DQMServices.Components.DQMEnvironment_cfi.dqmEnv.clone()
+dqmEnvL1TEMU.subSystemFolder = 'L1TEMU'
+
+# DQM Offline Step 1 cfi/cff imports
+from DQMOffline.L1Trigger.L1TRate_Offline_cfi import *
+from DQMOffline.L1Trigger.L1TSync_Offline_cfi import *
+from DQMOffline.L1Trigger.L1TEmulatorMonitorOffline_cff import *  
+l1TdeRCT.rctSourceData = 'gctDigis'
```

Comment:
this change also needs to go to its appropriate cfi (though I don't see where it comes in)